lib/src/platform_client.dart

```diff
@@ -1480,6 +1462,36 @@ class PlatformClient {
           MessagingClientPubNub(_config.messagingKeys!, _userId, token);
     }
   }
+
+  @visibleForTesting
+  static Exception? getExceptionFromApiResponse({
```

Comment:
```suggestion
// separate class
enum ErrorCodes {
  invalidToken('SEC-001'),
  businessLoginRequired('AIRA-ACCESS-017'),
  deleteAccount('KN-UM-065');

  const ErrorCodes(this.code);

  final String code;
}
// in getExceptionFromApiResponse method

switch (ErrorCodes.fromCode(errorCode)) {
      case ErrorCodes.invalidToken:
        return const PlatformInvalidTokenException();
      case ErrorCodes.businessLoginRequired:
      // TODO: Handle this case.
      case ErrorCodes.deleteAccount:
      // TODO: Handle this case.
    }


  
  
  
```

Reply:
Nice! I like this! 👍 
I will update it If we going to accept this PR. Since the original issue is not happen any more.